### ISSUE TITLE
Text-to-speech and other audio accomodations for blind/Visually impaired players on Windows

### ISSUE THREAD

**Issue by blinddevotion55** [CleverRaven/Cataclysm-DDA#26885]
I am legally blind, and I enjoy this game greatly. However, due to my limited vision, I sometimes struggle to notice information as it comes up in the sidebar. I have tried using two screen-reader programs to try to read text as it appears in the sidebar, or to read descriptions of items in different inventory menus to no avail. Although a soundpack from ChestHole has hadded some audio clues, I feel the game could be made much more accessible by either adding more audio cues or increasing compatibility with screen readers or adding a basic TTS engine to the game.

**Describe the solution you'd like**  
TTS Function:
--Needs to read off information as it appears in the sidebar. "You hear a huge explosion." for example, would be read aloud when that text appears. 
--Needs to announce what sort of menu has been opened, such as the activation menu or the smart inventory, consumables, or any more specific "context menu" like for vehicles.
--Needs to read whatever text is highlighted or under the cursor when navigating windows with the arrow keys or numpad, including telling the player what key they can press to quickly access that inventory item in the future. Same thing for the look function that's bound to "x".
--Keybind to read description of an item when it comes up in the activation and inventory menus. 
-----Might be unbound by default to avoid upsetting/annoying sighted players.
--separate keybind to read all the description and stats info on the page. 
-----also unbound by default
--If messages related to a character's hunger, thirst, or other health do not pop up in the sidebar text scroll, add those in? (I have never seen it but I'm severely visually impared so I might have just missed it.)
--ways to convey information like weather, time of day, or health estimates if precise information isn't available to the player. For example, for daylight, "It is afternoon" or "It is sunset". For health it would be something like "Your leg is uninjured" or lightly/moderately/severely injured or crippled.

**Describe alternatives you've considered**  
Alternative: Immersive audio clues to add to soundpack mods.
--Noises when hunger/thirst/fatigue or any other survival stat are changed. Could be a TTS thing saying "You are hungry" or a sound of a stomach growling, for example. Might need male/female sounds depending on character's sex.
--noises when new hostile creatures are seen or disappear from sight.
--Music change if a hostile mob is attempting to attack you
--A sound to alert the player if they are noticed by a mob.
--Sounds to inform the player when skills, quests, or other information updates without a message in the text flow window.

**Additional context**  
I am attempting to use both the console and graphical tiles version with no success in using 3rd party TTS programs and screen readers. The ones I have tried are NVDA and Zoomtext.

**Comment by ZhilkinSerg** [comment]
We will try [espeak](https://github.com/espeak-ng/espeak-ng) for TTS.

By the way, are you playing on Windows? Both versions there are not true-console, so most (or all) screen readers would fail. Did you try console version on Linux? Screen readers should be working there.
